nix/pkgs/core.nix

```diff
@@ -25,44 +25,16 @@ rec {
 
     # run all the non-slow cargo tests
     cargo build --features 'build' -p holochain_wasm_test_utils
-    cargo test ''${CARGO_TEST_ARGS:-} --workspace --exclude holochain --lib --tests --profile fast-test -- --nocapture
+    cargo test ''${CARGO_TEST_ARGS:-} --workspace --features slow_tests,test_utils,build_wasms,db-encryption --lib --tests --profile fast-test -- --nocapture
   '';
 
   hcStandardTestsNextest = writeShellScriptBin "hc-test-standard-nextest" ''
     set -euxo pipefail
     export RUST_BACKTRACE=1
 
-    # run all the non-slow cargo tests
+    # run all the cargo tests
     cargo build --features 'build' -p holochain_wasm_test_utils
-    cargo nextest ''${CARGO_NEXTEST_ARGS:-run --test-threads=2} --workspace --exclude holochain --lib --tests --cargo-profile fast-test
-  '';
-
-  hcSlowTests = writeShellScriptBin "hc-test-slow" ''
-    set -euxo pipefail
-    export RUST_BACKTRACE=1
-
-    # alas, we cannot specify --features in the virtual workspace
-    # run the specific slow tests in the holochain crate
-    cargo test ''${CARGO_TEST_ARGS:-} -p holochain --features slow_tests,test_utils,build_wasms,db-encryption --profile fast-test -- --nocapture
-  '';
-
-  hcSlowTestsNextest = writeShellScriptBin "hc-test-slow-nextest" ''
-    set -euxo pipefail
-    export RUST_BACKTRACE=1
-
-    # alas, we cannot specify --features in the virtual workspace
-    # run the specific slow tests in the holochain crate
-    cargo nextest ''${CARGO_NEXTEST_ARGS:-run --test-threads=2} -p holochain --features slow_tests,test_utils,build_wasms,db-encryption --cargo-profile fast-test
-  '';
-
-  hcSlowTestsIter = writeShellScriptBin "hc-test-slow-iter" ''
-    set -euo pipefail
-    export RUST_BACKTRACE=1
-
-    for i in `seq 1 ''${1}`; do
-      echo -n "$i: "
-      time env RUST_TEST_THREADS=$i hc-test-slow > /dev/null 2>&1
-    done
+    cargo nextest ''${CARGO_NEXTEST_ARGS:-run --test-threads=2} --workspace --features slow_tests,test_utils,build_wasms,db-encryption --lib --tests --cargo-profile fast-test
```

Comment:
could we remove the `slow_tests` feature altogether?

Reply:
I think @neonphog likes to have that there so that we can locally run a subset of tests relatively quickly during development. @neonphog care to chime in?

Reply:
@maackle @steveeJ - I haven't been able to run even the non-slow ci tests locally in over a year, it takes too much time and eventually runs out of memory anyways. Merging all the tests together will make no difference to me : )

Reply:
@neonphog have you tried running it with the nextest command? could you see what happens for you locally when you checkout this branch and run the following?

```
nix-shell --pure --run "ulimit -n 65000; hc-test-standard-nextest"
```

the result should match CI which is currently 3 tests failing consistently on this PR.

Reply:
@steveeJ - it did finish, so there's that : )

```
Summary [  56.486s] 569 tests run: 569 passed, 17 skipped
```

but... yeah, still not going to be a part of my regular dev iteration workflow:

```
real    9m39.096s
user    123m38.055s
sys     5m58.680s
```

Reply:
although... the summary only says 56 seconds, that'd be something i could handle... why did it take 9 minutes?

Reply:
apparently it just felt the need to rebuild everything again, so my target dir is now twice the size, but after that it runs in the 56 seconds : )

Reply:
fyi, the `ulimit` doesn't seem to be necessary on my system

Reply:
thanks @neonphog. i think this is because the test commands use a custom cargo profile:

https://github.com/holochain/holochain/blob/33f26d8f08e3c61f7981cd0a7441d479386da17a/Cargo.toml#L49-L58

so if you don't use this profile for your other builds it'll recompile